server/pulp/server/db/model/consumer.py

```diff
@@ -279,6 +279,7 @@ class UnitProfile(Model):
     collection_name = 'consumer_unit_profiles'
     unique_indices = (
         ('consumer_id', 'content_type'),
+        ('id'),
```

Comment:
@hao-yu , could you elaborate why it is needed?
I believe all models have a unique index for ids https://github.com/pulp/pulp/blob/2-master/server/pulp/server/db/model/base.py#L56

Reply:
The class rewrites indices, rather than extending them.
Could you extend the parent indices to make it as it was supposed to be? Thanks.
`unique_indices = Model.unique_indices + (('consumer_id', 'content_type'),)`